scheds/rust/scx_rusty/src/main.rs

```diff
@@ -157,6 +157,19 @@ struct Opts {
     #[clap(short = 'f', long, action = clap::ArgAction::SetTrue)]
     fifo_sched: bool,
 
+    /// Enable IO wait boosting.
+    #[clap(long, action = ::clap::ArgAction::SetTrue)]
+    iowait_boost: bool,
+
+    /// Maximum number of IO wait requests to boost frequency. After reaching the limit CPU
+    /// frequency will not be boosted for tasks with outstanding IO wait requests.
+    #[clap(long, default_value = "128")]
+    iowait_boost_max_ios: u32,
+
+    /// Maximum duration of IO wait boosting in milliseconds.
+    #[clap(long, default_value = "50")]
+    iowait_boost_max_dur_ms: u32,
```

Comment:
Just out of curiosity, did you see any kind of system performance degradation if we don't have these throttling knobs in place?